src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php

```diff
@@ -94,32 +96,30 @@ public function testSendWithOptions()
             ->method('getStatusCode')
             ->willReturn(200);
 
-        $content = <<<JSON
-            {
-                "ok": true,
-                "result": {
-                    "message_id": 1,
-                    "from": {
-                        "id": 12345678,
-                        "first_name": "YourBot",
-                        "username": "YourBot"
-                    },
-                    "chat": {
-                        "id": 1234567890,
-                        "first_name": "John",
-                        "last_name": "Doe",
-                        "username": "JohnDoe",
-                        "type": "private"
-                    },
-                    "date": 1459958199,
-                    "text": "Hello from Bot!"
-                }
-            }
-JSON;
+        $content = [
+            'ok' => true,
+            'result' => [
+                'message_id' => 1,
+                'from' => [
+                    'id' => 12345678,
+                    'first_name' => 'YourBot',
+                    'username' => 'YourBot',
+                ],
+                'chat' => [
+                    'id' => 1234567890,
+                    'first_name' => 'John',
+                    'last_name' => 'Doe',
+                    'username' => 'JohnDoe',
+                    'type' => 'private',
+                ],
+                'date' => 1459958199,
+                'text' => 'Hello from Bot!',
+            ],
+        ];
 
         $response->expects($this->once())
             ->method('getContent')
-            ->willReturn($content)
+            ->willReturn(json_encode($content))
```

Comment:
maybe would better to use `Symfony\Component\HttpClient\Response\JsonMockResponse` ?

Reply:
Like this?

Reply:
Yes :)